src/super_gradients/training/models/conversion.py

```diff
@@ -74,6 +75,10 @@ def convert_to_onnx(
         raise FileNotFoundError(f"Could not find destination directory {out_path} for the ONNX file.")
     torch_onnx_export_kwargs = torch_onnx_export_kwargs or dict()
     prep_model_for_conversion_kwargs = prep_model_for_conversion_kwargs or dict()
+
+    if get_param(prep_model_for_conversion_kwargs, "input_size") is None:
+        prep_model_for_conversion_kwargs["input_size"] = (1, *input_shape)
```

Comment:
Do we want to check that `input_size` (when it's present in prep_model_for_conversion_kwargs) equals the input_shape?
In other words, that prep_model_for_conversion gets the same spatial dimensions that we use of ONNX expor? Looks like a legit check for me

Reply:
Sure, added a check in latest commit.